packages/web/src/pages/upload-page/components/EditPageNew.tsx

```diff
@@ -78,35 +89,88 @@ export const EditPageNew = (props: EditPageProps) => {
     [onContinue, setTracks, tracks]
   )
 
-  const isMultiTrack = tracks.length > 1
-
   return (
     <Formik<TrackEditFormValues>
       initialValues={initialValues}
       onSubmit={onSubmit}
       validationSchema={EditTrackSchema}
     >
-      {() => (
-        <Form>
-          <div className={cn(layoutStyles.row, layoutStyles.gap2)}>
-            <div className={styles.editForm}>
-              <TrackMetadataFields playing={false} />
-              <TrackModalArray />
-              <PreviewButton playing={false} onClick={() => {}} />
-            </div>
-            {isMultiTrack ? <MultiTrackSidebar tracks={tracks} /> : null}
-          </div>
-          <div className={styles.continue}>
-            <HarmonyButton
-              variant={HarmonyButtonType.PRIMARY}
-              text='Continue'
-              name='continue'
-              iconRight={IconArrow}
-              className={styles.continueButton}
-            />
-          </div>
-        </Form>
-      )}
+      {TrackEditForm}
     </Formik>
   )
 }
+
+const TrackEditForm = (props: FormikProps<TrackEditFormValues>) => {
+  const { values } = props
+  const isMultiTrack = values.trackMetadatas.length > 1
+
+  return (
+    <Form>
+      <div className={cn(layoutStyles.row, layoutStyles.gap2)}>
+        <div className={styles.formContainer}>
+          {isMultiTrack ? <MultiTrackHeader /> : null}
+          <div className={styles.trackEditForm}>
+            <TrackMetadataFields playing={false} />
+            <TrackModalArray />
+            <PreviewButton playing={false} onClick={() => {}} />
+          </div>
+          {isMultiTrack ? <MultiTrackFooter /> : null}
+        </div>
+        {isMultiTrack ? <MultiTrackSidebar /> : null}
+      </div>
+      <div className={styles.continue}>
+        <HarmonyButton
+          variant={HarmonyButtonType.PRIMARY}
+          text='Continue'
+          name='continue'
+          iconRight={IconArrow}
+          className={styles.continueButton}
+        />
+      </div>
+    </Form>
+  )
+}
+
+const MultiTrackHeader = () => {
+  const [{ value: index }] = useField('trackMetadatasIndex')
+  const [{ value: trackMetadatas }] = useField('trackMetadatas')
+
+  return (
+    <div className={styles.multiTrackHeader}>
+      <Text variant='title' size='xSmall'>
+        {messages.multiTrackCount(index + 1, trackMetadatas.length)}
+      </Text>
+    </div>
+  )
+}
+
+const MultiTrackFooter = () => {
+  const [{ value: index }, , { setValue: setIndex }] = useField(
+    'trackMetadatasIndex'
+  )
+  const [{ value: trackMetadatas }] = useField('trackMetadatas')
+
+  const goPrev = useCallback(() => {
+    setIndex(Math.max(index - 1, 0))
+  }, [index, setIndex])
+  const goNext = useCallback(() => {
+    setIndex(Math.min(index + 1, trackMetadatas.length - 1))
+  }, [index, setIndex, trackMetadatas.length])
+
+  return (
+    <div className={cn(styles.multiTrackFooter, layoutStyles.row)}>
+      <HarmonyButton
```

Comment:
i think the colors need to be subdued when you cant go prev or next, ie if we on first track, prev should be subdued (disabled probably is sufficient)
<img width="820" alt="Pasted Graphic" src="https://github.com/AudiusProject/audius-client/assets/8230000/aacb0945-94c4-4399-bfd9-369a6d42087f">